js/components/urlBar.js

```diff
@@ -281,6 +275,9 @@ class UrlBar extends ImmutableComponent {
     this.clearSearchEngine()
     this.detectSearchEngine(e.target.value)
     this.keyPressed = false
+    if (e.target.value.length === 0) {
```

Comment:
just to be paranoid-safe, check that `e.target.value` exists before calling `e.target.value.length`